### ISSUE TITLE
[DOC] Opensearch 2.9 being installed when following 2.8 (current latest) Debian install doc

### ISSUE THREAD

**Issue by williamtrelawny** [opensearch-project/documentation-website#4618]
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [ ] Add new documentation
- [X] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 The Debian install docs for 2.8 (current latest) result in Opensearch 2.9 being installed. Which according to your Github [Releases](https://github.com/opensearch-project/OpenSearch/releases) page, isn't even GA yet.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
```
user@host:$ sudo apt-get install opensearch
  Reading package lists... Done
  Building dependency tree... Done
  Reading state information... Done
  The following NEW packages will be installed:
    opensearch
  0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
  Need to get 803 MB of archives.
  After this operation, 1,036 MB of additional disk space will be used.
  Get:1 https://artifacts.opensearch.org/releases/bundle/opensearch/2.x/apt stable/main amd64 opensearch amd64 2.9.0 [803 MB]
  Fetched 803 MB in 28s (29.0 MB/s)
  Selecting previously unselected package opensearch.
  (Reading database ... 36156 files and directories currently installed.)
  Preparing to unpack .../opensearch_2.9.0_amd64.deb ...
  Running OpenSearch Pre-Installation Script
  Unpacking opensearch (2.9.0) ...
  Setting up opensearch (2.9.0) ...
```

**Comment by hdhalter** [comment]
Hi @williamtrelawny - Thanks for reporting this! OpenSearch version 2.9 is releasing today, so there may have been a slight delay with syncing everything up. If you go to the documentation page now, you'll see that we've published the 2.9 content: https://opensearch.org/docs/latest

**Comment by williamtrelawny** [comment]
Oh wow just my perfect timing then haha. Sorry for the noise!
